tools/publish-verification-info.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+verification_dir="$1"
+img="$1/verification.raw"
+ip="$2"
+
+if [ "$verification_dir" == "" ] || [ "$ip" == "" ]; then
+  echo "Usage ./publish_verification_info.sh <verification_dir> <server_ip>"
+  exit
+fi
+
+size=$(fdisk -l "$img" | grep raw5 | awk '{print $2}')
+mkdir /tmp/verification_mnt
```

Comment:
Do we suppose the scrip running only once?
Otherwise, it will give an error here as the directory already exists.

Reply:
If you see line 34 the dir is being deleted.

Reply:
Ah, sorry! Missed it!